Sentry SDK (#784)
fewlinesco/connect-account: README.adoc

We are using https://docs.sentry.io/[Sentry] to monitor production and review app's exceptions raised.

> Thanks to the https://docs.sentry.io/platforms/javascript/guides/nextjs/[Sentry Next.js SDK], the implementation is now way cleaner. We have added a wrapper around the `withSentry` middleware to be able to use is it with SSR.

Reviewer: ```suggestion
We're using https://docs.sentry.io/platforms/javascript/guides/nextjs/[Sentry Next.js SDK] and have added a wrapper around the `withSentry` middleware to be able to use is it with SSR.
```

Reading a README telling about an improvement is weird imho, since we don't read the previous versions.

Author: Make sense.
